### DISCUSSION TITLE
Archinstall of 2024.11.01 (ISO) broken

### DISCUSSION THREAD

**Discussion by TechGameGeek** [archlinux/archinstall#2864] [General]
Hi folks,
it seems like that the Archiso  2024.11.01 has some problems within the archinstall script. It is running into an error while doing its "python-things" ;-)

![archfailed](https://github.com/user-attachments/assets/d91d33cf-9d28-4652-96c5-5b4350f13a15)

Using an older ISO or cloning the latest archinstall from git after booting the latest ISO 2024.11.01 solves it.

I can upload the installer-log later (not enough time now).

Thanks for your great work!

**Comment by svartkanin** [comment]
This has already been foxed in the latest release

**Reply by TechGameGeek** [reply]
Thanks for the reply -- sorry I should have thought twice, that I have to report it to the arch-devs (ISO Creators).
